i18n.py

```diff
@@ -2,9 +2,12 @@
 import json
 import os
 
+from pathlib import Path
 
-def load_language_list(language):
-    with open(f"./lib/i18n/{language}.json", "r", encoding="utf-8") as f:
+
+def load_language_list(language, base_dir):
+    language_file_path = os.path.join(base_dir, f"lib/i18n/{language}.json")
```

Comment:
I believe `base_dir / f"lib/i18n/{language}.json"` should work.
Maybe with `str(language_file_path)` later.

Reply:
Done